starboard/tools/testing/test_runner.py

```diff
@@ -184,6 +196,28 @@ def Kill(self):
   def Join(self):
     self.runner_thread.join()
 
+  def _Run2(self):
```

Comment:
Should this be `def _Run2(self) -> None:` to make it clear that the return code does not get propagated through return value ?

Reply:
Done.